index.html

```diff
@@ -2672,7 +2672,8 @@ <h5 class="organizers-title">Meet the Organizers of W/S/C 2017</h5>
   </footer>
 
   <!-- ## LOAD JAVASCRIPTS ## -->
-  <script src="./assets/js/2.2.2.jquery.min.js" type="text/javascript"></script>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+  <!-- <script src="./assets/js/2.2.2.jquery.min.js" type="text/javascript"></script> -->
```

Comment:
I clicked around to make sure the version bump didn't hurt us, looks ok to me! Maybe @deweydell should also have a look since I think she's added the most JS functionality...

Once you get her 👍 , delete the comment (line 2676) and :shipit: !